core/shared/src/main/scala/org/polyvariant/colorize.scala

```diff
@@ -121,12 +150,43 @@ object colorize {
 
   private[colorize] case class CurrentColor(value: String) extends AnyVal
 
+  sealed trait ColorMode
+
+  object ColorMode {
+    private[colorize] case object Ansi extends ColorMode
+    private[colorize] case object TrueColor extends ColorMode
```

Comment:
Pretty sure these need to be public to allow users to choose in the render config